cg/meta/workflow/raredisease.py

```diff
@@ -19,3 +25,40 @@ def __init__(
         pipeline: Pipeline = Pipeline.RAREDISEASE,
     ):
         super().__init__(config=config, pipeline=pipeline)
+
+    @property
+    def root(self) -> str:
+        return self.config.raredisease.root
+
+    def write_panel(self, case_id: str, content: list[str]) -> None:
+        """Write the gene panel to case dir."""
+        out_dir = Path(self.root, case_id)
+        out_dir.mkdir(parents=True, exist_ok=True)
+        WriteFile.write_file_from_content(
+            content="\n".join(content),
+            file_format=FileFormat.TXT,
+            file_path=Path(out_dir, f"gene_panels{FileExtensions.BED}"),
+        )
+
+    @staticmethod
+    def get_aggregated_panels(customer_id: str, default_panels: set[str]) -> list[str]:
+        """Check if customer should use the gene panel master list
+        and if all default panels are included in the gene panel master list.
+        If not, add gene panel combo and OMIM-AUTO.
+        Return an aggregated gene panel."""
+        master_list: list[str] = GenePanelMasterList.get_panel_names()
+        if customer_id in GenePanelMasterList.collaborators() and default_panels.issubset(
+            master_list
+        ):
+            return master_list
+        all_panels: set[str] = add_gene_panel_combo(default_panels=default_panels)
+        all_panels.add(GenePanelMasterList.OMIM_AUTO)
+        return list(all_panels)
+
+    def get_gene_panel(self, case_id: str, genome_build: str = GENOME_BUILD_37) -> list[str]:
+        """Create and return the aggregated gene panel file."""
```

Comment:
will rare-diseases be use for both mip-rna/mip-dna? If so, the get_gene_panel function likely should not have a genome_build preset as they seem to differ for mip-rna/mip-dna above.